plugins/AuctionMaster/config.yml

```
@@ -0,0 +1,335 @@
+previous-page-item-name: '&aPage %page-number%'
+previous-page-item-lore:
+  - '&7Click para ir a'
+  - '&7la pagina de antes.'
+
+next-page-item: '262'
+#Use %page-number% for the current page's number
+next-page-item-name: '&aPage %page-number%'
+next-page-item-lore:
+  - '&7Click para ir a'
+  - '&7la siguiente pagina.'
+
+starting-menu-name: '&8Menu de pujas'
+close-menu-item: '166'
+close-menu-item-name: '&cCerrar Menu'
+close-menu-item-lore:
+  - '&7Cerrar Menu'
+
+browsing-menu-name: '&8Navegador de subastas'
+browsing-menu-item: '41'
+browsing-menu-item-name: '&6Navegador de subastas'
+browsing-menu-item-lore:
+  - '&7Busca artículos a la venta por'
+  - '&7jugadores alrrededor del server!'
+  - ''
+  - '&7Items los items ofrecidos aqui son para'
+  - '&6auction&7, lo que significa que tienes que'
+  - '&7haz la mejor oferta para conseguirlos!'
+  - ''
+  - '&eClic para navegar!'
+
+view-bids-menu-name: '&8Visor de ofertas de subasta'
+view-bids-menu-item: '396'
+#Use %total-bids% for the number of auctions the player has bid on
+#Use %total-top-bid% for the number of auctions the player holds the top bid on
+view-bids-menu-item-name: '&aVer ofertas'
+view-bids-menu-item-lore:
+  with-bids:
+    - '&7Has puesto &a%total-bids% bids&7 en'
+    - '&7apuestas pendientes.'
+    - ''
+    - '&7Mantienes la oferta superior en'
+    - '&7(&a%total-top-bid%&7/&a%total-bids%&7) de estas'
+    - '&7pujas.'
+    - ''
+    - '&eClick para ver!'
+  without-bids:
+    - '&7No has realizado ninguna oferta'
+    - '&7en apuestas pendientes.'
+    - ''
+    - '&7Haz ofertas por artículos y haz click '
+    - '&7aqui para verlas!'
+    - ''
+    - '&cNo tienes ninguna oferta!'
+
+no-bids-message: '&cNo tienes ninguna oferta!'
+
+delivery-menu-name: '&8Menú de entrega'
+use-delivery-system: true
+
+#This is the item for deliveries in the main menu
+delivery-menu-item-name: '&aEntregas'
+delivery-menu-item: '54'
+delivery-menu-item-lore:
+  - '&7Aquí vienen los artículos y'
+  - '&7monedas mandados por admins!'
+
+delivery-have-message: '&aTienes entregas!'
+
+#This is the item for deliveries in the delivery menu
+delivery-item:
+  no-delivery-name: '&cNo hay entrega'
+  no-delivery-lore:
+    - '&7No tienes ninguna'
+    - '&7entrega '
+    - '&7para reclamar. :('
+  #Use %coins% for the amount of coins the player has in their delivery
+  #Use %items-count% for the amount of items the player has in their delivery
+  yes-delivery-name: '&aReclamar entrega'
+  yes-delivery-lore:
+    - '&7Tienes una entrega!'
+    - ''
+    - '&7La entrega contiene:'
+    - '&7 - &6%coins% monedas'
+    - '&7 - &e%items-count% articulos'
+    - ''
+    - '&aClick para reclamar!'
+
+delivery-claimed-message: '&aHas reclamado tu recompensa!'
+not-enough-space-delivery: '&cNo tienes suficiente espacio para reclamar toda la entrega!'
+no-delivery-to-claim: '&cNo tienes ninguna entrega que reclamar!'
+
+delivery-alert-join-message: '&6[Auction] &fTienes entregas no reclamadas!'
```

```suggestion
delivery-alert-join-message: '&6[Auction] &f¡Tienes entregas no reclamadas!'
```
Creo que sería mejor poniendo "¡X entregas no reclamadas!"